### ISSUE TITLE
Hard-coded dataset split paths in `evaluate.py` and `evaluate_files.py`.

### ISSUE THREAD

**Issue by cisprague** [gcorso/DiffDock#60]
Hi, thanks for making the code public.

The paths for the dataset splits seem to be hard-coded in:
1. https://github.com/gcorso/DiffDock/blob/8e853d6b14fb57baf90fa8529349117439f06819/evaluate_files.py#L29
2. https://github.com/gcorso/DiffDock/blob/8e853d6b14fb57baf90fa8529349117439f06819/evaluate_files.py#L30
3. https://github.com/gcorso/DiffDock/blob/8e853d6b14fb57baf90fa8529349117439f06819/evaluate.py#L166

IMO, it would be easier to use this framework for user-defined splits if there was an option to provide the path.

Also, what is the description of these dataset splits?

**Comment by HannesStark** [comment]
Thanks we made the changes!
The first path you mention contains the names of all the pdbbind complexes in our test set.
The second path contains the names of all the pdbbin complexes in the test set whose receptor does not appear in the training data
